src/main/java/com/checkmarx/ast/wrapper/CxWrapper.java

```diff
@@ -216,6 +217,21 @@ public List<Project> projectList(String filter) throws IOException, InterruptedE
         return Execution.executeCommand(withConfigArguments(arguments), logger, Project::listFromLine);
     }
 
+    public ScanResult ScanVorpal(String fileSource, boolean vorpalLatestVersion) throws IOException, InterruptedException, CxException {
+        this.logger.info("Fetching Vorpal scanResult");
+
+        List<String> arguments = new ArrayList<>();
+        arguments.add(CxConstants.CMD_SCAN);
+        arguments.add(CxConstants.SUB_CMD_VORPAL);
+        arguments.add(CxConstants.FILE_SOURCE);
```

Comment:
add verification that it's not null (the file source). it can throw form the CLI to have a flag without a value

Reply:
Didn't we say that the CLI should not return error in this case because it's a valid case? (return empty response in this case)